src/v2/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -581,7 +581,10 @@ export class ShippingRoute extends Component<ShippingProps, ShippingState> {
     const isArtaShipping = this.isArtaShipping()
     const isContinueButtonDisabled = isCommittingMutation
       ? false
-      : isArtaShipping && !shippingQuoteId && !this.isCreateNewAddress()
+      : isArtaShipping &&
+        !!shippingQuotes &&
+        shippingQuotes.length > 0 &&
+        !shippingQuoteId
```

Comment:
minor, non-blocking: I think we could probably get rid of this ternary operator:
```js
const isContinueButtonDisabled = !isCommittingMutation &&
							  	 isArtaShipping &&
						         !!shippingQuotes &&
						         shippingQuotes.length > 0 &&
						         !shippingQuoteId
```